### PR TITLE
pythonPackage.ovirt-engine-sdk: init at 4.3.3

### PR DESCRIPTION
**Pull request by jficz**
###### Motivation for this change

Ansible ovirt_* modules require this lib

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1411,6 +1411,12 @@
     githubId = 411324;
     name = "Carles Pagès";
   };
+  cptMikky = {
```
Comment (jonringer) [inline maintainers/maintainer-list.nix]: should be a separate commit before the package addition:

```
maintainers: add cptMikky
```

File: pkgs/development/python-modules/ovirt-engine-sdk/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, pkgs }:
```
Comment (jonringer) [inline pkgs/development/python-modules/ovirt-engine-sdk/default.nix]: please import libxml directly
```suggestion
{ lib, buildPythonPackage, fetchPypi, libxml2 }:
```

File: pkgs/development/python-modules/ovirt-engine-sdk/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+  buildInputs = with pkgs.python37Packages; [ 
+    pycurl 
+    six 
+    enum34
+    pkgs.libxml2
+  ];
+
+  prePatch = ''
+    substituteInPlace setup.py --replace "/usr/include/libxml2" "${pkgs.libxml2.dev}/include/libxml2"
```
Comment (jonringer) [inline pkgs/development/python-modules/ovirt-engine-sdk/default.nix]: in case dev output gets removed (unlikely)
```suggestion
    substituteInPlace setup.py --replace "/usr/include/libxml2" "${lib.getDev libxml2}/include/libxml2"
```

File: pkgs/top-level/python-packages.nix [hunk 4]
```diff
@@ -834,6 +834,8 @@ in {
 
   outcome = callPackage ../development/python-modules/outcome {};
 
+  ovirt-engine-sdk = callPackage ../development/python-modules/ovirt-engine-sdk {};
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix]:
```suggestion
  ovirt-engine-sdk = callPackage ../development/python-modules/ovirt-engine-sdk {
    inherit (pkgs) libxml2;
  };
```

**Comment by jonringer** [comment]
and squash the license change into the package commit as well please :)

File: pkgs/development/python-modules/ovirt-engine-sdk/default.nix [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+buildPythonPackage rec {
+  pname = "ovirt-engine-sdk-python";
+  version = "4.3.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0hz7rcz7l391n8ixcwvqw3iq5bx26fk9l5c6xxymv036zm3r9pnh";
+  };
+
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/ovirt-engine-sdk/default.nix]: Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.

File: pkgs/development/python-modules/ovirt-engine-sdk/default.nix [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+  buildInputs = with pkgs.python37Packages; [
```
Comment (FRidh) [inline pkgs/development/python-modules/ovirt-engine-sdk/default.nix]: Python packages are not often in `buildInputs`. This would imply the Python packages in this list are only needed during build-time and not run-time. Is that correct?

File: pkgs/development/python-modules/ovirt-engine-sdk/default.nix (hunk 5 above)
Comment (cptMikky) [reply pkgs/development/python-modules/ovirt-engine-sdk/default.nix]: honestly, I just copy-pasted this derivation from some other package. Not sure what the tests do but I can enable them

File: pkgs/development/python-modules/ovirt-engine-sdk/default.nix (hunk 6 above)
Comment (cptMikky) [reply pkgs/development/python-modules/ovirt-engine-sdk/default.nix]: you're right, not correct - I misunderstood the buildInputs behavior, will fix, thanks

**Comment by cptMikky** [comment]
closing in favor of [TBA]
